plugin.video.catchuptvandmore/resources/language/English/strings.po

```diff
@@ -0,0 +1,137 @@
+# Kodi Media Center language file
```

Comment:
Please use the new layout for language folders (since this is for krypton):
`/resources/language/resource.language.en_gb/strings.po`
`/resources/language/resource.language.fr_fr/strings.po`
`/resources/language/resource.language.it_it/strings.po`

See: http://kodi.wiki/view/Language_support